commands_test.go

```diff
@@ -4029,6 +4029,28 @@ var _ = Describe("Commands", func() {
 
 	})
 
+	Describe("SlowLog", func() {
+
+		It("returns slow query result", func() {
+
+			var (
+				CONFIGSLOWKEY string = "slowlog-log-slower-than"
+			)
+			old := client.ConfigGet(ctx, CONFIGSLOWKEY).Val()
+			client.ConfigSet(ctx, CONFIGSLOWKEY, "0")
+			defer client.ConfigSet(ctx, CONFIGSLOWKEY, old[1].(string))
+			_, e := client.Eval(ctx, "redis.call('slowlog','reset')", []string{}).Result()
```

Comment:
I wonder if you use `Eval` to execute custom command? go-redis supports `client.Do(ctx, "slowlog", "reset")` to achieve the same.

Reply:
I want to keep the same code-style, which all test case in this file  use `client.Eval` to execute the command :)